### ISSUE TITLE
System.ArgumentException: Key was already exists. Serializing my class

### ISSUE THREAD

**Issue by Miro382** [neuecc/Utf8Json#91]
Hi,
I receiving this exception when I try serialize my custom class:
Exception thrown: 'System.ArgumentException' in Utf8Json.dll
System.ArgumentException: Key was already exists. Key:System.Byte[]
   at Utf8Json.JsonFormatterResolverExtensions.GetFormatterWithVerify[T](IJsonFormatterResolver resolver)
   at Utf8Json.Formatters.Good_Teacher_DataStoreFormatter1.Serialize(JsonWriter& , DataStore , IJsonFormatterResolver )
   at Utf8Json.JsonSerializer.ToJsonString[T](T value, IJsonFormatterResolver resolver)
   at Utf8Json.JsonSerializer.ToJsonString[T](T value)

Thanks

**Comment by OlegNadymov** [comment]
I have similar problem. Any ideas?

**Comment by jensbrak** [comment]
Yup. Got that one too. A generic class with a dictionary based on enums in my case.
